contracts/gateway/AxelarAmplifierGateway.sol

```diff
@@ -138,14 +141,19 @@ contract AxelarAmplifierGateway is IAxelarAmplifierGateway {
             revert CommandAlreadyExecuted(commandId);
         }
 
+        bool applyRotationDelay = msg.sender != _storage().rotationOperator;
```

Comment:
Maybe we actually expose `rotateSigners` in the `BaseWeightedMultisig` and add the functionality of rotation there? It's already returning `isLatestSigners`, so the contract is kinda aware of that.

Also it could use it's own variant of domain separator and batch could not need a commandID